### PR TITLE
Rename SGPC3 methods

### PR DESCRIPTION
**Pull request by rnestler**
The last one of the large name refactoring.

### CODE REVIEW

File: sgpc3/sgpc3.c [hunk 1]
```diff
@@ -705,71 +705,43 @@ s16 sgpc3_get_serial_id(u64 *serial_id) {
...
- */
-s16 sgpc3_iaq_init16() {
-    return sgpc3_run_profile_by_number(PROFILE_NUMBER_IAQ_INIT16);
-}
-
-/**
- * sgpc3_iaq_init64() - reset the SGP's internal IAQ baselines using accelerated
- *                    startup time of 64s
+ * Note: legacy command for old SGPC3 fs<=5: initializes algo, for only TVOC,
+ * with preheating for 64s. Does not work for the ULP power mode!
```
Comment (abrauchli) [inline sgpc3/sgpc3.c]: I would formulate it as `does not work in conjunction with sgpc3_set_power_mode` ULP is never used as term..

Comment (rnestler) [reply sgpc3/sgpc3.c]: Hmm in the `sgpc3_set_power_mode` documentation we state:
```
 * @power_mode: Power mode to set:
 *              0: Ultra low power mode (30s measurement interval)
 *              1: (Default) Low power mode (2s measurement interval)
```

We could just add something like ULP (ultra low power) and LP (low power) there.

Comment (abrauchli) [reply sgpc3/sgpc3.c]: yes that'd work, but IMHO it's easier to match against the function if someone is not sure..

Comment (rnestler) [reply sgpc3/sgpc3.c]: Fixed: See https://github.com/Sensirion/embedded-sgp/pull/51/commits/4a3818e87d2c86efe5f21ebee01a41329a01be43.

**Pull request merged**